### PR TITLE
Vav entity type hereworks 18.10.24

### PR DESCRIPTION
**Pull request by Ciaran-Oflannagain**

### CODE REVIEW

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 1]
```diff
@@ -333,6 +333,7 @@ CO2C:
   opt_uses:
   - high_zone_air_co2_concentration_alarm
   - failed_zone_air_co2_concentration_alarm
+  - zone_air_co2_concentration_deadband_setpoint
```
Comment (shambergoldstein) [inline ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:336]: This needs to be added as a telemetry field. Please change to `zone_air_deadband_co2_concentration_setpoint`

File: ontology/yaml/resources/HVAC/entity_types/VAV.yaml [hunk 2]
```diff
@@ -56,6 +56,16 @@ VAV_SD_HLSAFS:
   - SD
   - HLSAFS
 
+VAV_SD_HLSAFS_CO2C:
```
Comment (shambergoldstein) [inline ontology/yaml/resources/HVAC/entity_types/VAV.yaml:59]: Please confirm that there is no temperature control.

**Review by shambergoldstein — CHANGES_REQUESTED** [review]

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml (hunk 1 above)
Comment (Ciaran-Oflannagain) [reply ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml]: updated

File: ontology/yaml/resources/HVAC/entity_types/VAV.yaml (hunk 2 above)
Comment (Ciaran-Oflannagain) [reply ontology/yaml/resources/HVAC/entity_types/VAV.yaml:59]: Hi Shamber - no temp control, control based on the co2 setpoint

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml (hunk 1 above)
Comment (shambergoldstein) [reply ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml]: still needs to be added to telemetry fields

Comment (Ciaran-Oflannagain) [reply ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml]: @shambergoldstein - apologies forgot to commit telemetry yaml, updated now

**Review by shambergoldstein — APPROVED** [review]

**Comment by shambergoldstein** [comment]
@Ciaran-Oflannagain apoligies, it looks like the validator is failing with this subfield order, please try `zone_air_co2_deadband_concentration_setpoint` and see if that resolves the error.

**Comment by Ciaran-Oflannagain** [comment]
@shambergoldstein - apologies, updated to  zone_air_co2_deadband_concentration_setpoint

**Pull request merged**